ecommerce-iosWidget/WidgetView/LargeWidgetView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  LargeWidgetView.swift
+//  ecommerce-iosWidget
+//
+//  Created by Bliss on 14/5/21.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct LargeWidgetView: View {
+
+    var body: some View {
+        ZStack {
+            VStack {
+                Spacer()
+                Rectangle()
+                    .fill(Color.gray100)
+                    .frame(maxWidth: .infinity, maxHeight: 182.0)
+            }
+            VStack {
+                // TODO: large widget top part, replace below line
+                Spacer(minLength: 147.0)
+                // end replace
+
+                Spacer(minLength: 25.0)
+                ItemsGridView(items: ["1", "2", "3", "4", "5", "6", "7"])
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                Spacer()
```

Comment:
I don't think we need this `Spacer()` because we have `.padding()` in line 31. Do you think so?